CHANGELOG.md

```diff
@@ -14,6 +14,18 @@
 
 - The API `PATCH` method for `/machine-config` can be now used to change
   `track_dirty_pages` on aarch64.
+- Replaced `mem_file_path` parameter for `/snapshot/load` API by `mem_backend`.
```

Comment:
Before going any deeper here, we need to ensure the changes that we make to the API are backwards compatible. In this regard, just replacing `mem_file_path` with another parameter is not enough. We need to first make `mem_file_path` optional, deprecate it, then add this new `mem_backend` parameter as optional. In the absence of the new parameter, the legacy one must be present, as per the old specification where it was required.

Reply:
Snapshots are currently GA, so it's not a `requirement` to ensure backwards compatibility. I agree, however, that it would be better for our users to be compatible with the old functionality.